docs/README.rst

```diff
@@ -0,0 +1,107 @@
+.. _readme:
+
+cron-formula
+============
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/cron-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/cron-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+Formula to set up and configure cron
```

Comment:
I'm not blocking this whole PR because of this!  But since I'm requesting changes, may as well request this as well:

```suggestion
Formula to set up and configure cron.
```